sections/04_study.md

```diff
@@ -144,47 +144,47 @@ and some configurations of tools like OneCodex[ref] and LMAT[ref].  While one
 cannot identify which reads were mapped back to an organism using relative
 abundance estimators, they can be useful for faster comparative and other
 downstream analyses.   Newer methods hope to classify reads and estimate
-relative abundances at faster rates [Vervier] and as of this writing, there
+relative abundances at faster rates [@tag:Vervier] and as of this writing, there
 are more than 70 metagenomic taxonomic classifiers in existence.  Besides
 binning and classification of species, there is functional identification and
-annotation of sequence reads [Yok,Soueidan]. However, the focus on
+annotation of sequence reads [@tag:yok @tag:Soueidan]. However, the focus on
 taxonomic/functional annotation is just the first step.  Once organisms are
 identified, there is the interest in understanding the interrelationship
-between these organisms and host/environment phenotypes [Guetterman].  One of
+between these organisms and host/environment phenotypes [@tag:Guetterman].  One of
 the first attempts was a survey of supervised classification methods for
-microbes->phenotype classification [Knights], followed by similar studies
-that are more massive in scale [Stratnikov, Segata].  There have been
-techniques that bypass the taxonomic classification step altogether [ Ding et al]
-, (sequence composition to phenotype classification).  Also, researchers have
-looked into how feature selection can improve classification [Liu, Segata],
+microbes->phenotype classification [@tag:Knights], followed by similar studies
+that are more massive in scale [@tag:Stratnikov @tag:Segata].  There have been
+techniques that bypass the taxonomic classification step altogether [@tag:Ding],
+(sequence composition to phenotype classification).  Also, researchers have
+looked into how feature selection can improve classification [@tag:Liu @tag:Segata],
 and techniques have been proposed that are classifier-independent
 [Ditzler,Ditzler].
```

Comment:
@gailrosen not sure what to do with these Ditzler references as `tags.tsv` contains Ditzler, Ditzler2, Ditzler3